pkg/columns/formatter/textcolumns/scaler.go

```diff
@@ -297,6 +297,8 @@ func (tf *TextColumnsFormatter[T]) AdjustWidthsToContent(entries []*T, considerH
 		totalWidth += column.calculatedWidth
 	}
 
+	tf.buildFillString()
+
```

Comment:
Could you please describe why it is needed?

Reply:
Sure - it takes the longest possible string and builds a string with whitespace to copy from. This is used to quickly fill strings to a specific length without having to do a for loop every time. But whenever the calculatedWidths are adjusted, it needs to be rebuilt - and that was missing here.